docs/settings.md

```diff
@@ -0,0 +1,204 @@
+
+### Naming
+
+#### Volume Folder Naming
+
+The naming format for the volume folder (folder structure allowed).  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+
+For example, `{series_name} ({year})` will come out as "Invincible Iron Man (2019)".
+
+#### File Naming
+
+The naming format for the file itself.  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+- {issue_comicvine_id}
+- {issue_number}
+- {issue_title}
+- {issue_release_date}
+- {issue_release_year}
+
+For example, `{series_name} ({year}) Volume {volume_number} Issue {issue_number}` would come out as "Invincible Iron Man (2019) Volume 01 Issue 001.cbz".
+
+#### File Naming For TPB
+
+The naming format for the file itself (if it's a TPB).  
+Available variables are:
+
+- {series_name}
+- {clean_series_name}
+- {volume_number}
+- {comicvine_id}
+- {year}
+- {publisher}
+
+For example, `{series_name} ({year}) Volume {volume_number} TPB` would come out as "Invincible Iron Man (2019) Volume 01 TPB.cbz"
+
+#### Issue Padding
+
+This defines how issue numbers are 'padded' (3 digits, 2 digits, etc.).  
+This is useful in the case of file sorting, as many operating systems sort _alphabetically_, instead of alphanumerically. To them, because 1 is lower than 2, "11" comes before "20".  
+To get around this, we 'pad' the numbers with leading zeroes. "1" becomes"001", "11" becomes "011", and "2" becomes "002" - resulting in a more accurate sort.
+
+Options are:
+
+- x (no padding, i.e. 1,2,...10)
+- 0x (2 digits, i.e. 01, 02...10)
+- 00x (3 digits, i.e. 001, 002...010...099,100)
+
+#### Volume Padding
+
+This defines how volume numbers are 'padded' (3 digits, 2 digits, etc.).  
+This is useful in the case of file sorting, as many operating systems sort _alphabetically_, instead of alphanumerically. To them, because 1 is lower than 2, "11" comes before "20".  
+To get around this, we 'pad' the numbers with leading zeroes. "1" becomes"001", "11" becomes "011", and "2" becomes "002" - resulting in a more accurate sort.
+
+Options are:
+
+- x (no padding, i.e. 1,2,...10)
+- 0x (2 digits, i.e. 01, 02...10)
+- 00x (3 digits, i.e. 001, 002...,010...099,100)
+
+### Unzipping
+
+Unzip downloads will extract zip files after downloading.  
+This is useful for 'pack' style downloads, where the download contains multiple issues.
+
+### Root Folders
+
+A 'root folder' is the path that Kapowarr will use to create volume folders and store files.  
+If you've mounted the container using the example paths, this will likely be `/content`.  
+_Note: at least one of these must be set before you are able to add any volumes to monitor, as without it Kapowarr cannot know where to put the files._
+
+## Download
+
+### Download Location
+
+#### Direct Download Temporary Folder
```

Comment:
A lot of info here overlaps with what is told in the Setup After Installation page. I think it's best to constantly refer to here from that page. This is relevant for the following topics on the SAI page: Port, Authentication, ComicVine API key, Root Folders (replace text of Settings page with the one of the SAI page, then refer in SAI to Settings page), Download Folder, Service Preference and Credentials.

That is basically the whole page, so maybe just create a list of links. Like a "You should take a look at below settings (in listing order) to properly set up Kapowarr" and then just a bullet list of the settings and a link to their explanation on the settings page. Do this for the whole page up to "Building up a library".

Reply:
This one seems like a pretty decent-sized rewrite - will work towards this tomorrow.